examples/petsc/include/petscversion.h

```diff
@@ -0,0 +1,8 @@
+#ifndef petsc_version
+#define petsc_version
```

Comment:
This might be legacy, but this isn't our namespace. We can define `libceed_petsc_version_h` or whatever, but not `petsc_version`.